blog-page.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adoptadog - Blog</title>
+    <link rel="stylesheet" type="text/css" href="./styles/styles.css">
+  </head>
+
+  <body>
+    <header>
+      <div class="header-main">
+        <div class="logo-and-name">
+          <img src="./images/adoptadog-logo.svg" alt="White outline of a dog sitting on its hind legs">
+          <h1>Seattle's premiere dog adoption agency</h1>
+        </div>
+
+        <aside class="header-cart">
+          <img src="./images/cart-icon.svg" alt="White outline of a shopping cart">
+          <p>$1234.56</p>
+          <a href="checkout-page.html">Checkout</a>
+        </aside>
+      </div>
+
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="adopt-page.html">Dogs</a></li>
+          <li><a href="blog-page.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <h2>Adoptadog Blog</h2>
+        <article class="blog">
```

Comment:
one suggestion would be to use flexbox wrap for your blog class. this will allow your text to wrap below the image on smaller screen sizes. Right now your text gets squished on small screens